content/assets/blog/phoenix-case-study/index.md

```diff
@@ -0,0 +1,137 @@
+   streams of social media data in real-time.
+
+2. **Limited technical expertise**: Many peacebuilders aren’t data scientists, so any AI-driven
+   platform had to be intuitive and accessible.
+
+3. **Multi-language support**: Conflicts often span regions with multiple languages, including
+   those that traditional analytics tools rarely handle well.
+
+4. **Open-source requirement**: Build Up wanted a project open to contributors worldwide,
+   facilitating collaboration and continuous improvement.
+
+***
+
+## **The solution: Phoenix—An AI platform for peacebuilders**
+
+Working closely with Build Up, datavaluepeople created Phoenix: a powerful, user-friendly system
+that addresses each challenge in a single, comprehensive platform.
+
+### **Modern web console**
+
+Built with React and Next.js, Phoenix’s web console offers a straightforward interface accessible
+from any device with an internet connection. It allows peacebuilders to easily configure analyses,
+explore data sources, and review insights without advanced technical skills.
+
+### **Authentication and dashboarding**
+
+Phoenix’s custom authentication system ensures secure access, while its Superset-based dashboards
+make data visualization highly flexible. Users can build interactive charts and graphs that help
+surface trends or spikes in conflict-related discussions—providing clarity on community sentiment
+and emerging risks.
+
+### **Data scraping integration**
+
+By integrating with data-scraping platforms like Apify, Phoenix streamlines large-scale data
+collection from social media and other online sources. It lets users define specific parameters for
+what data to capture, helping them focus on the most relevant content while honouring platform and
+data privacy policies.
+
+### **Classification systems for all languages**
+
+A standout feature is Phoenix’s multilingual classification engine, which manages high- and
+low-resource languages alike. This ensures peacebuilders can quickly identify posts of
+interest—crucial when sifting through huge volumes of potentially conflict-related material.
+
+***
+
+## **How it works**
+
+1. **Gather**: Peacebuilders specify keywords, locations, or other criteria for collecting data
+   from social media and other sources.
+
+2. **Classify**: Phoenix’s machine learning models process the incoming data, identifying critical
+   topics, trends, and sentiments.
+
+3. **Explore**: Users start with a standard dashboard and tailor visualizations to track regional
+   tensions or other priorities.
+
+4. **Collaborate & share**: Being open-source, Phoenix allows contributors worldwide to enhance its
+   capabilities—ensuring continued evolution and impact.
+
+***
+
+## **The results: A global impact on peacebuilding**
+
+Since launch, Phoenix has proven invaluable for peacebuilders worldwide, yielding several
+noteworthy outcomes:
+
+1. **Global reach**: Versions v1 and v2 of Phoenix have supported more than 50 peacebuilders,
```

Comment:
Surely it's at more than 50 by now? If you include anyone that has ever touched the platform or even looked at a dashboard (i.e. the UNDP Lebanon people who were getting the reports) etc etc?

Reply:
0ed3735